test/cases/ease.out.css

```diff
@@ -1,6 +1,3 @@
 body {
-  -ms-transition: 200ms cubic-bezier(0.680, -0.550, 0.265, 1.550);
-  -moz-transition: 200ms cubic-bezier(0.680, -0.550, 0.265, 1.550);
-  -webkit-transition: 200ms cubic-bezier(0.680, -0.550, 0.265, 1.550);
   transition: 200ms cubic-bezier(0.680, -0.550, 0.265, 1.550);
```

Comment:
Since the browser prefixes have been removed, should we add autoprefixer in as a utility of styl?